packages/app/src/providers/h5grove/__snapshots__/h5grove-api.test.ts.snap

```diff
@@ -499,27 +681,45 @@ exports[`test file matches snapshot 1`] = `
   },
   {
     "name": "ascii_vlen_empty",
-    "rawType": "|O",
+    "rawType": {
+      "class": 3,
+      "cset": 0,
+      "dtype": "|O",
+      "size": 8,
+      "vlen": true,
+    },
     "shape": null,
     "type": {
-      "charSet": "UTF-8",
+      "charSet": "ASCII",
```

Comment:
The new `rawType` provided by h5grove allows for more accurate parsing of string encoding.